src/content/1.7/development/page-reference/back-office/product/form/_index.md

```diff
@@ -28,50 +28,207 @@ This form combines different sub form types, each one is handled by dedicated CQ
 
 | Form type                | Field      | Description                                              | CQRS Command                                           |
 |:-------------------------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------|
-| **BasicInformationType** | `basic` | Contains the basic product information of the Product | `AddProductCommand` and `UpdateBasicInformationCommand` |
-| **DescriptionType** | `description`    | Contains the description of the Product | `UpdateDescriptionCommand` |
-| **ShortcutType** | `shortcut`    | Contains shortcut for prices and quantity of the Product | `UpdatePricesCommand`, `UpdateStockCommand` and `UpdateOptionsCommand` |
-| **RelationshipsType**    | `relationships` | Contains relationships of the product | N/A |
-| **ShortcutType**         | `shortcuts`     | Contains shortcut fields editable in other sub forms but accessible in first tab easily | N/A |
-| **TypeaheadProductPackCollectionType** | `pack_items` | List of products (for Pack of product) | `AddProductToPackCommand` `UpdateProductPackCommand` `RemoveProductFromPachCommand` |
+| **BasicInformationType** | `basic` | Contains the basic product information of the Product | `AddProductCommand` and `UpdateProductBasicInformationCommand` |
+| **DescriptionType** | `description`    | Contains the description of the Product | `UpdateProductBasicInformationCommand` |
+| **ShortcutType** | `shortcut`    | Contains shortcut for prices, quantity and reference of the Product | `UpdateProductPriceCommand`, `UpdateProductStockCommand` and `UpdateProductOptionsCommand` |
+| **TypeaheadProductPackCollectionType** | `pack_items` | List of products (for Pack of product) | `UpdateProductPackCommand` `CleanProductFromPackCommand` |
```

Comment:
There is no update action for a pack, we can only remove the packed product, so i would say:
- AddProductToPackCommand (https://github.com/PrestaShop/PrestaShop/pull/19827) 
- RemoveProductFromPackCommand

Reply:
If we want some improvements, we could also add additional command like `updatePackedProductQuantityCommand`

Reply:
Just a note to close this discussion.
As discussed in slack, implemented UpdateProductPackCommand which resets the packs association completely to a new provided pack.